src/network-client.ts

```diff
@@ -49,6 +76,21 @@ export function getAlgoNodeConfig(network: 'testnet' | 'mainnet', config: 'algod
   }
 }
 
+/** Returns the Account configuration
+ *
+ * @param accountName account name
+ */
+export function getAccountConfig(accountName: string): AccountConfig {
```

Comment:
Also wonder if this should be called `getAccountConfigFromEnvironment` just so it is clear